### PR TITLE
Adding support for s390x and ppc64le for multi-arch image build

### PR DESCRIPTION
**Pull request by satyamg1620**
## Description
Provided support for s390x and ppc64le to create multi-arch image

## How Has This Been Tested?
For creating multi-arch build image, use command ``` make docker-buildx ```.  This will build multi-arch images and will push into your provided image repository

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [ ] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ include ${ENV_FILE}
 export $(shell sed 's/=.*//' ${ENV_FILE})
 endif
 
-CONTAINER_BUILDER?=podman
+CONTAINER_BUILDER?=docker
```
Comment (lucferbux) [inline Makefile:13]: We use `podman` as our default container builder due to license limitation of the docker engine nowadays. We would want to keep it that way in our automations, so we encourage overriding with docker in the local environment filest to the people that need it

File: Makefile [hunk 2]
```diff
@@ -30,6 +30,21 @@ build:
 
 ##################################
 
+# Build multi-arch image
+
+PLATFORMS ?= linux/s390x,linux/amd64,linux/ppc64le
```
Comment (lucferbux) [inline Makefile:35]: And so far this works great, tested with an M1 mack and dockerx and it seems to be working just fine

**Review by lucferbux — CHANGES_REQUESTED** [review]
Hi Satyam,

Thanks for the PR, tested the command and it seems to be working great, we don't want to change the default container builder, we encourage you to parametrize with docker the command if you want to use it, but don't change the default builder engine.

Thanks a lot!

**Review by dpanshug — CHANGES_REQUESTED** [review]
@satyamg1620 Can you pls squash all the commits

**Comment by satyamg1620** [comment]
@dpanshug, @christianvogt , @lucferbux Can you please review the requested changes?

**Comment by satyamg1620** [comment]
@dpanshug , @christianvogt  , @lucferbux  Can you please review the requested changes?

**Review by lucferbux — APPROVED** [review]
/lgtm

Thanks for the contribution, tested on my machine and it seems to work great!

Screenshot 2024-04-16 at 17 40 31

**Pull request merged**